packages/suite/src/actions/suite/metadataLabelingActions.ts

```diff
@@ -529,6 +568,297 @@ export const addMetadata =
             return result.success;
         });
 
+/**
+ *
+ * @returns files in storage provider split into [[...current][...old][...renamedOld]].
+ */
+const getMetadataFiles =
```

Comment:
1. we shall use `createThunk` everywhere 
2. new code shall provide ALL there generic parameters to `createThunk`